Makefile

```diff
@@ -768,7 +772,7 @@ $(TARGET): $(OBJS)
 ifdef RELEASE
   ifndef DEBUG_SYMBOLS
     ifneq ($(BACKTRACE),1)
-	$(STRIP) $(TARGET)
+      $(STRIP) $(TARGET)
```

Comment:
It seems something is not right here.

Reply:
Yep, my bad; that has to remain a tab, not spaces.  Sorry, didn't test in release build.